Apps/W1/EDocumentConnectors/SignUp/test/src/IntegrationHelpers.Codeunit.al

```diff
@@ -0,0 +1,67 @@
+namespace Microsoft.EServices.EDocumentConnector.SignUp;
+
+
+codeunit 148196 IntegrationHelpers
+{
+    internal procedure SetAPIWith200Code()
+    begin
+        this.SetAPICode('/signup/200');
+    end;
+
+    internal procedure SetAPIWith500Code()
+    begin
+        this.SetAPICode('/signup/500');
+    end;
+
+    internal procedure SetAPICode(Path: Text)
+    var
+        SignUpConnectionSetup: Record SignUpConnectionSetup;
+    begin
+        SignUpConnectionSetup.Get();
+        SignUpConnectionSetup."Service URL" := this.SetMockServiceUrl(Path);
+        SignUpConnectionSetup.Modify(true);
+    end;
+
+    internal procedure SetCommonConnectionSetup()
+    var
+        SignUpConnectionSetup: Record SignUpConnectionSetup;
+        SignUpAuthentication: Codeunit SignUpAuthentication;
+    begin
+        SignUpConnectionSetup.Get();
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Root App ID", this.DummyId());
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Root Secret", this.DummyId());
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Root Tenant", this.DummyId());
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Client ID", this.DummyId());
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Client Secret", this.DummyId());
+        SignUpAuthentication.StorageSet(SignUpConnectionSetup."Client Tenant", this.ClientTenantId());
+
+        SignUpConnectionSetup."Authentication URL" := this.SetMockServiceUrl('/%1/oauth2/token');
+        SignUpConnectionSetup."Environment Type" := SignUpConnectionSetup."Environment Type"::Test;
+        SignUpConnectionSetup.Modify(true);
+    end;
+
+    internal procedure SetMockServiceUrl(Path: Text): Text[250]
+    begin
+        exit('http://localhost:8080' + Path);
```

Comment:
I'm unsure about the tests.

The tests here seem to use a Mock Service.
This isn't possible in pipelines.
@Groenbech96 do you still allow this in new tests?
The tests might be helpful, but they would have to be disabled in the CI.

It would be better if there were some tests that uses another way for mocking responses.
e.g. via Interfaces (check out the RestClient Module in the system application).

Reply:
Yes we use this in the internal build. 
We are building http client mocking support in the language, but it is not yet released.